packages/analytics-js-integrations/__tests__/integrations/Appcues/appcues.test.js

```diff
@@ -0,0 +1,163 @@
+import { Appcues } from '../../../src/integrations/Appcues';
+
+const appcuesConfig = {
+  accountId: '812354',
+  eventFilteringOption: 'disable',
+  whitelistedEvents: [],
+  blacklistedEvents: [],
+  oneTrustCookieCategories: [],
+  nativeSdkUrl: { web: '' },
+  useNativeSDK: { web: true },
+};
+const destinationInfo = {
+  areTransformationsConnected: false,
+  destinationId: 'sample-destination-id',
+};
+const testEmail = 'test@email.com';
+
+const mockAppcuesSDK = () => {
+  window.Appcues = {
+    identify: jest.fn(),
+    track: jest.fn(),
+    page: jest.fn(),
+  };
+};
+
+beforeEach(() => {
```

Comment:
Is there any reason we have to do this in `beforeEach` rather than `beforeAll` ?

Reply:
Yes it was not getting initialised, checked while debugging so did it this way